com.ibm.wala.cast.java.ecj/build.gradle

```diff
@@ -52,7 +52,7 @@ tasks.named('test') {
 	// temporarily turn off some tests on JDK 9+
 //	if (JavaVersion.current() <= JavaVersion.VERSION_1_8) {
 		// ensure the command-line driver for running ECJ works
```

Comment:
Maybe we can remove these commented lines while we are editing here?  Also the ones starting at line 56

Reply:
If we want to remove these lines, then we should do so separately, since that's unrelated to the narrowly focused purpose of this pull request.